Geometry/HcalTowerAlgo/test/HcalCellParameterDump.cc

```diff
@@ -21,13 +21,13 @@ class HcalCellParameterDump : public edm::one::EDAnalyzer<> {
   void endJob() override {}
 
 private:
-  static const int detMax_ = 4;
+  static constexpr int detMax_ = 4;
   int subdet_;
 };
 
 HcalCellParameterDump::HcalCellParameterDump(const edm::ParameterSet& iConfig) {
-  subdet_ = std::min(detMax_, std::max(iConfig.getParameter<int>("SubDetector"), 1));
-  subdet_ = std::min(detMax_, std::max(subdet_, 1));
+  subdet_ = std::min(HcalCellParameterDump::detMax_, std::max(iConfig.getParameter<int>("SubDetector"), 1));
```

Comment:
@smuzaffar Why is the `HcalCellParameterDump::` qualification necessary? There is no other `detMax_` that could cause confusion, and the constructor certainly has access to its own data members.

Reply:
you are right @cvuosalo , I started with adding the namespace but then it turned out the `constexpr` should fix the issue. I have updated the PR